app/models/edition.rb

```diff
@@ -33,6 +35,10 @@ def import_csv(file)
     false
   end
 
+  def month_and_year
```

Comment:
shouldn't this be under **private**?

Reply:
Nope, it's fine here because it's called from the controller